ui/src/views/submissions/submission-forms/forms/SubmissionEditForm.tsx

```diff
@@ -619,7 +639,13 @@ class SubmissionEditForm extends React.Component<Props, SubmissionEditFormState>
                         </Upload>
                       }
                       bodyStyle={{ padding: '0' }}
-                    ></Card>
+                    >
+                      <Input
+                        placeholder="Alt text"
```

Comment:
question: so just to double check, this only really applies to the thumbnail and primary file?
Not seeing anything about the additional files area. Which is fine, just wanted to make sure I was understanding it correctly.

Reply:
It applies to the primary file and all additional files. The thumbnail doesn't get an alt text input because no site lets you specify alt text for a thumbnail.

![image](https://github.com/mvdicarlo/postybirb-plus/assets/13625824/6b42c120-2591-4ab2-beed-c632d84495c9)